frontend/src/libs/components/button/button.tsx

```diff
@@ -1,10 +1,17 @@
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  className?: string | undefined;
 };
 
-const Button: React.FC<Properties> = ({ type = 'button', label }) => (
-  <button type={type}>{label}</button>
+const Button: React.FC<Properties> = ({
+  type = 'button',
+  label,
+  className = '',
+}: Properties) => (
```

Comment:
```suggestion
}) => (
```
FC<Properties> already handles this